snuba/datasets/storages/metrics.py

```diff
@@ -20,59 +25,102 @@
 )
 from snuba.utils.streams.topics import Topic
 
-buckets_columns = ColumnSet(
-    [
-        Column("org_id", UInt(64)),
-        Column("project_id", UInt(64)),
-        Column("metric_id", UInt(64)),
-        Column("metric_type", String()),
-        Column("timestamp", DateTime()),
-        Column("tags", Nested([Column("key", UInt(64)), Column("value", UInt(64))])),
-        Column("set_values", Array(UInt(64))),
-        Column("materialization_version", UInt(8)),
-        Column("retention_days", UInt(16)),
-        Column("partition", UInt(16)),
-        Column("offset", UInt(64)),
-    ]
-)
+PRE_VALUE_COLUMNS: Sequence[Column[SchemaModifiers]] = [
+    Column("org_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("metric_id", UInt(64)),
+    Column("timestamp", DateTime()),
+    Column("tags", Nested([Column("key", UInt(64)), Column("value", UInt(64))])),
+]
 
-buckets_storage = WritableTableStorage(
+POST_VALUE_COLUMNS: Sequence[Column[SchemaModifiers]] = [
+    Column("materialization_version", UInt(8)),
+    Column("retention_days", UInt(16)),
+    Column("partition", UInt(16)),
+    Column("offset", UInt(64)),
+]
+
+sets_buckets = WritableTableStorage(
     storage_key=StorageKey.METRICS_BUCKETS,
     storage_set_key=StorageSetKey.METRICS,
     schema=WritableTableSchema(
-        columns=buckets_columns,
+        columns=ColumnSet(
+            [
+                *PRE_VALUE_COLUMNS,
+                Column("set_values", Array(UInt(64))),
+                *POST_VALUE_COLUMNS,
+            ]
+        ),
         local_table_name="metrics_buckets_local",
         dist_table_name="metrics_buckets_dist",
```

Comment:
Will send a followup PR to change the name of this table that was meant to contain all types of metrics.